modules/labs-react/common/README.md

```diff
@@ -16,3 +16,41 @@ yarn add @workday/canvas-kit-labs-react/common
 
 For more detailed information on this component, please refer to the
 [storybook documentation](https://workday.github.io/canvas-kit/?path=/docs/labs-common)
+
+## useThemeRTL
+
+This is a way to automatically transform CSS, applied using either Emotion's css attribute or by a
+style attribute, when there is a RTL theme in use. Instead of authoring two sets of CSS objects, one
+for each language direction. Now you can author the LTR version and this hook will automatically
+create the RTL counterpart for you on the fly. This works for both object and string template
+Emotion methods.
+
+For convenience we also export a ComponentStyles type if you want to organize your styles all
+together.
+
+### Usage
+
+```tsx
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import {ComponentStyles, useThemeRTL} from '@workday/canvas-kit-labs-react/common';
+const styles: ComponentStyles = {
+  panel: {
+    margin: '8px',
+  },
+  top: {
+    top: '8px',
+  },
+  red: {
+    color: 'red',
+  },
+};
+export const HelloWorld = () => {
+  const {themeRTL} = useThemeRTL();
+  return (
+    <div css={themeRTL(styles.panel, styles.top)} style={themeRTL(styles.red)}>
```

Comment:
```suggestion
  const {themeRTL, theme} = useThemeRTL();
  const panelStyles = {
    ...themeRTL(styles.panel),
    backgroundColor: theme.canvas.palette.neutral.main, 
  };
  // You can also pass `panelStyles` to the `style` prop, if preferred.
  return (
    <div css={panelStyles} style={{...styles.top, ...styles.red}}>
```

A couple things here as well:

* show `theme` being provided as well and why you might want to use it so it's not hidden
* passing bidirectional styles to themeRTL and separating out the rest


I'm very open to feedback on this usage though. If it makes more sense to leave it as:

```tsx
<div css={themeRTL(styles.panel, styles.top)} style={themeRTL(styles.red)}>
```

I'm good with that too. 👍🏻

Reply:
> passing bidirectional styles to themeRTL and separating out the rest

I might avoid doing this, seems error prone and then you have to think about what needs to be transformed. Like the example of using theme